deploy/deployment_dev.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
I believe that if we use OpenShift `DeploymentConfig`, we won't need the `Service` definition below

Reply:
Are you sure?

Reply:
We don't need any service for operators, as it's not publicly exposed to the outside the cluster and also it's not rest service